lib/components/STable.vue

```diff
@@ -338,20 +366,27 @@ function getCell(key: string, index: number) {
   .STable.has-footer & {
     border-radius: 0;
   }
+
+  .block {
+    margin-bottom: -5px;
```

Comment:
Not sure why this is needed. There is a consistent gap after the last row of ~5.09px. Can't understand where it's coming from 🥲.